cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java

```diff
@@ -587,11 +587,11 @@ private static void writeDataAlignedByDevice(
           AtomicReference<Boolean> isFail = new AtomicReference<>(false);
 
           // read data from record
-          for (String measurement : headerNameMap.values()) {
-            String value = record.get(measurement);
+          for (Map.Entry<String, String> headerName : headerNameMap.entrySet()) {
+            String value = record.get(headerName.getValue());
```

Comment:
It's better to use two variables to indict the headerName.getValue() and headerName.getKey(), this would be more readble.

eg：

`Map.Entry<String, String> headerNameEntry : headerNameMap.entrySet()

// headerNameWithoutType is equal to headerName if the CSV coloum do not have data type.
String  headerNameWithoutType =  headerNameEntry.getKey();
String  headerName =  headerNameEntry.getValue(); `

Reply:
fix~